README.md

```diff
@@ -274,8 +274,7 @@ under utilized frequently or for a short period of time. By default, `numberOfNo
 This strategy finds nodes that are under utilized and evicts pods from the nodes in the hope that these pods will be 
 scheduled compactly into fewer nodes.  Used in conjunction with node auto-scaling, this strategy is intended to help 
 trigger down scaling of under utilized nodes.
-This strategy **must** be used with the scheduler strategy `MostRequestedPriority`. The parameters of this strategy are 
-configured under `nodeResourceUtilizationThresholds`.
+The parameters of this strategy are configured under `nodeResourceUtilizationThresholds`.
```

Comment:
@JaneLiuL it's still important to mention that this has to be used with the right scheduler config, like in https://github.com/kubernetes-sigs/descheduler/issues/668#issuecomment-988167924 and https://github.com/kubernetes-sigs/descheduler/issues/668#issuecomment-989887089